nbs/docs/Prediction_intervals_in_forecasting_models.ipynb

```diff
@@ -0,0 +1,1394 @@
+{
```

Comment:
<p>I think you can get rid of the "Table of Contents" here by referencing the sections by their name directly in the cell above, something like:</p><ol><li>[Introduction](#introduction)</li><li>[Prediction intervals in forecasting models](#prediction-intervals-in-forecasting-model)</li><li>...</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/mlforecast/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>